hal/inc/interrupts_hal.h

```diff
@@ -83,6 +84,10 @@ typedef struct hal_interrupt_extra_configuration_t {
       uint8_t keepHandler  : 1;
     };
   };
+  struct {
+      bool appendHandler;
```

Comment:
Perhaps this can be added as a flag into the above flags/union? The unused bits are 0 right now in every call.